From 7352be11359e628352d47a53a8b3c239231a964a Mon Sep 17 00:00:00 2001
From: Donald Chan <hoiho@lab126.com>
Date: Mon, 21 Sep 2020 10:50:13 -0700
Subject: [PATCH] Migrate to PyCryptodome

---
 scripts/pem_to_pub_c.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/pem_to_pub_c.py b/scripts/pem_to_pub_c.py
index 3a896a3..d3f0e50 100755
--- a/scripts/pem_to_pub_c.py
+++ b/scripts/pem_to_pub_c.py
@@ -21,8 +21,8 @@ def get_args():
 
 def main():
     import array
-    from Crypto.PublicKey import RSA
-    from Crypto.Util.number import long_to_bytes
+    from Cryptodome.PublicKey import RSA
+    from Cryptodome.Util.number import long_to_bytes
 
     args = get_args()
 
